### ISSUE TITLE
Cannot install plugins or themes

### ISSUE THREAD

**Issue by zaifastafa** [urre/wordpress-nginx-docker-compose#41]
It asks for FTP credentials everytime I try to install a plugin or theme. 

![image](https://user-images.githubusercontent.com/24492269/82249573-76132680-9963-11ea-9bb9-d467a4067832.png)

**Comment by urre** [comment]
@zaifastafa did you read the readme? This setup uses Composer to manage plugins and themes.

**Comment by araujoantonio** [comment]
Hi, I have a similar question:

I bought a theme, it comes in a zip file, I don't think it's possible to install it through composer.

How should I proceed to install it? When I upload it then hit install, I see this error. I don't know it's related to this issue.
`get_current_page was called incorrectly. Current page retrieval should be called on or after the current_screen hook.`

 I know it's a noobie question. But if you have any ideas on how to fix I would be very grateful.

**Comment by gerontius** [comment]
I'm using the repo as a reference so I may be wrong but I would think that your host volume is the key here. Add your files to your host and they should be available to the container. Im used to working composer on drupal and I wouldnt be using composer on the container (or production environments). If you dont want to use composer then just download in a normal way to your host. Im not an expert on wordpress development so I could be wrong.

**Comment by urre** [comment]
@araujoantonio @gerontius Two ways:

1. You can enable plugin and theme updates and installation from the admin like this:
Edit `src/config/environments/development.php`

Set these two flags:
```
Config::define('DISALLOW_FILE_EDIT', false);
Config::define('DISALLOW_FILE_MODS', false);
```
Then you can update/install plugin/themes through wp-admin

2. You can also just drop in the theme in `src/web/app/themes`

**Comment by araujoantonio** [comment]
Hi @urre 

Thanks for your answer. I've tried to do what you suggested, but it didn't work. I'm still seeing the error message that I reported first (`get_current_page was called incorrectly. Current page retrieval should be called on or after the current_screen hook`) when I enter the front-end setup wizard for the theme.

For me just dropping the theme under `src/web/app/themes` is not a solution because it requires some sort of a build process (done by the setup wizard).

For now, I will go forward with the official docker image, but if you have any ideas on how to solve this I would love to hear.
